### ISSUE TITLE
No Collatrix output for Whale Body Condition Function

### ISSUE THREAD

**Issue by abaz-oceans** [cbirdferrer/collatrix#16]
Hi Clara, 

I'm attempting to use the Whale Body Condition Function in CollatriX and running into a snag. 

I got the same message screenshotted below whether I selected the csv containing the collated measurements outputted by CollatriX from the allIDs (measured in m) or PixelCount (pixels).  Does it matter which of the two files we select for the first input? I'm looking to calculate a BAI in pixels. 

Thank you
[2022_allIDs.csv](https://github.com/cbirdferrer/collatrix/files/11951871/2022_allIDs.csv)

![Capture](https://github.com/cbirdferrer/collatrix/assets/137197142/2e494383-17bf-49cc-912e-689290c7d82d)
[2022_PixelCount.csv](https://gi
[2022_allIDs.csv](https://github.com/cbirdferrer/collatrix/files/11951869/2022_allIDs.csv)
thub.com/cbirdferrer/collatrix/files/11951867/2022_PixelCount.csv)
[2022_allIDs.csv](https://github.com/cbirdferrer/collatrix/files/11951871/2022_allIDs.csv)

**Comment by cbirdferrer** [comment]
Hi! Hmm.. well if you got the same error on both then I don't think it's a PixCount error, although that might cause problems later since the headers are different. The BAI value should be the same ever when calculated from the meters version. From the error message, it looks like the problem is that your Animal_ID is numeric (2022) and the image name is a character, so I'd suggest converting the animal ID column to a character (maybe even just add a character prefix or something) and then trying again. Let me know how it goes!

**Comment by abaz-oceans** [comment]
Hi Clara, 

Thanks for your response. I corrected the Animal ID for the 2022 folder with a character prefix but received a new error screenshotted below. I also tried running the function on a PixelCount .csv where all the Animal ID's are just the name of the folder (ex., "Replicates") not separate names (i.e., "Whale1", "Whale2") and the Image names are like DJI_0026.jpeg, but got the same result attached.

I'm hoping I am one step closer if it is the same when trying all my different test folders.

All the best,
AB

![Capture](https://github.com/cbirdferrer/collatrix/assets/137197142/0c69a139-6c68-4f80-9cc9-38c40f911ea3)
